REVIEW_COPY.md

```diff
@@ -0,0 +1,312 @@
+# Drag & Drop without Draggables & Dropzones
+
+Drag & drop UIs have been around for a while, and so you would be forgiven for thinking they're a solved problem. Certainly there's high quality libraries to help us build drag & drop UIs, and these days we even have an official HTML5 drag & drop API! Why then is building drag & drop UIs so hard?
+
+Having built some drag & drop UIs I've come to the conclusion a big part of the problem is our choice of tools. Those tools are the draggables and dropzones most libraries provide for building drag & drop interactions. Not using draggables and dropzones to build UIs often simplifies their construction massively.
```

Comment:
I'm not sure what this actually means. Are you saying the problem is the draggables and dropzones? The last sentence is where I lost the thread.

Also small edit to more directly answer the question you propose in the previous paragraph.

```suggestion
I've come to the conclusion a big part of the problem is our choice of tools: the draggables and dropzones most libraries provide for building drag & drop interactions. Not using draggables and dropzones to build UIs often simplifies their construction massively.
```

Reply:
> I'm not sure what this actually means. Are you saying the problem is the draggables and dropzones? The last sentence is where I lost the thread.

I see how I'm being unclear here, thank you for flagging it!

How about:

> I've come to the conclusion a big part of the problem is our choice of tools: the draggables and dropzones most libraries provide for building drag & drop interactions. Draggables and dropzones are often the wrong abstraction, and choosing different tools can simplify construction of our drag & drop UIs massively.

Reply:
Yeah, lots clearer!

I think you could tighten it up further by removing " the draggables and dropzones most libraries provide for building drag & drop interactions"